### ISSUE TITLE
Missing Keys, Invalid Rows, Sample Violation Missing

### ISSUE THREAD

**Issue by corycundy** [sql-bi/VertiPaq-Analyzer#105]
I am running a VertiPaq Analyzer scan against a SQL Server 2016 tabular model of 1.65 GB and 42 tables. I am running this in DAX Studio 3.0.11 and TE3 3.10.1 with collect stats turned on in both.

In both I am showing RI issues in 15 of the 42 tables on the Tables tab. 

In DAX Studio, on the Relationships tab, none of the relationships show Missing Keys, Invalid Rows or Sample Violation Missing. In TE3, I get an error included below. My guess is that DAX Studio is getting the same exception, but swallowing it.

I'm sure this will be hard to reproduce in a non-client environment since it is likely data related, but wanted to log it. I don't know if there is any other detailed logging that I can provide.

<img width="379" alt="image" src="https://github.com/sql-bi/VertiPaq-Analyzer/assets/64218283/7e10db09-7ff8-4d4f-b47e-38c8d79949e3">

**Comment by marcosqlbi** [comment]
The error is an exception raised by the SSAS engine. It seems you have the latest version of DAX Studio, which is updated with the latest VPAX libraries. You are using an older version of TE that certainly has an older version of VPAX libraries, and it's possible there is some bug.
You might want to try [Bravo](https://bravo.bi) that also uses the latest version of VPAX libraries, if it works, then I would check whether you can update TE (probably the next release will have the same libraries as DAX Studio has).

**Comment by corycundy** [comment]
I upgraded Tabular Editor 3 to 3.12.0 which came out yesterday and received the same error as above. 

I tried using Bravo, but didn't see how I could connect it to an instance of SQL Server Analysis Services 2016.

After further digging, I was able to capture the VertiPaq queries with the All Queries feature of DAX Studio and found out it was the same exact issue that was discussed in a DAX Studio issue from two years ago.

https://github.com/DaxStudio/DaxStudio/issues/459
